phys2denoise/tests/test_rvt.py

```diff
@@ -0,0 +1,30 @@
+import peakdet
```

Comment:
I'm not sure if we want to include `peakdet` as a dependency to this repo. I think the assumption is that users have run `peakdet` _before_ `phys2denoise`, so they are separate. @smoia WDYT?

Reply:
That's the idea indeed, but I guess here the purpose is using `peakdet` to simulate a fake physiological file, right?

In that case (clever!), we can add `peakdet` as an extra dependency for tests ONLY.

(@tsalo don't be jealous, I gave you `pandas` as extra dependency in `phys2bids` :stuck_out_tongue_closed_eyes: )

Reply:
Btw, you _should_ add it to the dependency. If you want to know how, have a look at the setup.cfg file in `phys2bids`.

Reply:
I added it as a test dependency.